backend/routes/brandRoutes.js

```diff
@@ -1,14 +1,19 @@
 const express = require('express');
 const router = express.Router();
 const { signup, login, getProfile, updateProfile } = require('../controllers/brandController');
+const brandAuthenticationMiddleware = require('../middleware/brands/brandAuthenticationMiddleware');
+const brandSignUpValidate = require('../middleware/brands/validations/brandSignUpValidate');
+
+// Signup a brand
+router.post('/signup',brandSignUpValidate, signup);
 
 // Login as a brand
 router.post('/login', login);
 
 // Get brand's profile
-router.get('/profile', getProfile);
+router.get('/profile',brandAuthenticationMiddleware, getProfile);
 
 // Update brand's profile
-router.put('/profile', updateProfile);
+router.put('/profile',brandAuthenticationMiddleware, updateProfile);
```

Comment:
same as above, we should have authorization here as well right so that brand 1 shouldn't be able to update the data of brand2